cocos/audio/assets/clip.ts

```diff
@@ -33,6 +33,7 @@ import { Enum } from '../../core/value-types';
 import { AudioPlayer, PlayingState } from './player';
 import { AudioPlayerDOM } from './player-dom';
 import { AudioPlayerWeb } from './player-web';
+import { legacyGlobalExports } from '../../core/global-exports';
```

Comment:
The name is too long and not friendly to the package size (consider how frequent it's used), I suggest `legacyCC` instead

Reply:
OK, that's fine. By the way, the name of this kind of module variable would be uglified.

Reply:
Oh, I didn't know that